tests/secc/states/test_iso15118_2_states.py

```diff
@@ -334,3 +335,24 @@ async def test_power_delivery_set_hlc_charging(
         )
 
         self.comm_session.evse_controller.set_hlc_charging.assert_called_with(False)
+
+    @pytest.mark.parametrize(
+        "get_state_return_value, expected_next_state",
+        [
+            (CpState.C2, CurrentDemand),
```

Comment:
Would be good to check C1, D1 ends up in Terminate as well

Reply:
done